src/status_im/subs/wallet/activities.cljs

```diff
@@ -1,119 +1,165 @@
 (ns status-im.subs.wallet.activities
   (:require
     [native-module.core :as native-module]
-    [quo.foundations.resources :as quo.resources]
-    [quo.foundations.resources]
+    [quo.foundations.resources :as resources]
     [re-frame.core :as rf]
-    [status-im.contexts.wallet.common.activity-tab.constants :as constants]
+    [status-im.constants :as constants]
+    [status-im.contexts.wallet.common.activity-tab.constants :as activity-constants]
+    [utils.collection :as collection]
     [utils.datetime :as datetime]
-    [utils.hex :as utils.hex]
+    [utils.hex :as hex]
     [utils.money :as money]))
 
-(def precision 6)
+;; Constants
+(def ^:private precision 6)
 
-(rf/reg-sub
- :wallet/all-activities
- :<- [:wallet]
- :-> :activities)
+;; Conversion helpers
+(defn- hex->number
+  [hex-str]
+  (-> hex-str
+      hex/normalize-hex
+      native-module/hex-to-number
```

Comment:
status-go's hex-to-number has some precision issues for numbers with big amount of decimals, instead we should use money/from-hex which uses BigNumber